concent_api/concent_api/system_check.py

```diff
@@ -129,3 +153,27 @@ def storage_cluster_certificate_path_check(app_configs = None, **kwargs):  # pyl
         elif not os.path.isfile(certificate_path):
             errors.append(create_error_15_ssl_cert_path_is_not_a_file(certificate_path))
     return errors
+
+
+@register()
+def creation_new_chain_segment_time_check(app_configs  = None, **kwargs):  # pylint: disable=unused-argument
+    errors = []
+    creation_new_chain_segment_time = settings.CREATION_NEW_CHAIN_SEGMENT_TIME
```

Comment:
Please also add check at the beginning that it is even set, and it must be set if `PAYMENT_BACKEND` is set to sci. So:

```
if (
    not hasattr(settings, PAYMENT_BACKEND) or
    settings.PAYMENT_BACKEND != 'core.payments.sci_backend' and
):
    return errors (we don't want to validate if backend is not sci)
if (
    hasattr(settings, PAYMENT_BACKEND) and
    settings.PAYMENT_BACKEND == 'core.payments.sci_backend' and
    not hasattr(settings, CREATION_NEW_CHAIN_SEGMENT_TIME):
    some_error()
```